go.mod

```diff
@@ -57,7 +57,7 @@ require (
 	github.com/miekg/dns v1.1.66
 	github.com/mistifyio/go-zfs/v3 v3.1.0
 	github.com/mitchellh/copystructure v1.2.0
-	github.com/moby/buildkit v0.26.3
+	github.com/moby/buildkit v0.26.1-0.20251217224254-19a0de691e8a
```

Comment:
I was surprised that this worked and no replace rule was needed (because `v0.26.1-0.<date>` is lower than `v0.26.3`) but I guess it's because we don't have other dependencies using BuildKit.

Anyone using `github.com/moby/moby/v2` as dependency will have to add a replace though (but that module should not be used as dependency).